src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -458,6 +458,11 @@ private void OnUpdateGraphCompleted(AsyncTask task)
         /// </summary>
         public void Run()
         {
+            if (!RunSettings.RunEnabled)
+            {
+                return;
+            }
+
```

Comment:
There have been some comments about this in the summary. Hmmm, I will add some comments here as well.